src/ui/common/src/components/operators/operatorParametersOverview.tsx

```diff
@@ -51,6 +52,27 @@ const OperatorParametersOverview: React.FC<Props> = ({
           {(exParams as GoogleSheetsExtractParams).spreadsheet_id}
         </Typography>
       );
+    } else if ('query_serialized' in exParams) {
+      const mongoDbParams = exParams as MongoDBExtractParams;
+      return (
+        <Box>
+          <Typography variant="body2" color={textColor}>
+            <strong>table: </strong>
+            <code>{mongoDbParams.table}</code>
+          </Typography>
+          <Typography variant="body2" color={textColor}>
+            <strong>query: </strong>
+            <code>
```

Comment:
We have a CodeBlock component that you may want to consider using. Should be able to get JSON syntax highlighting with it too. 

This is more of a nice to have or something that can be addressed in another PR though. I will be changing the internals of the Codeblock component soon so that we use a library which doesn't emit so many .css files when the build finishes.

Reply:
let me try, this is a part of a small table cell so it might not look as good as we expected

Reply:
sounds good!

Reply:
Not bad, I think we can go with this:
<img width="929" alt="Screen Shot 2022-11-03 at 4 12 49 PM" src="https://user-images.githubusercontent.com/10411887/199852403-14f82748-7803-41a7-a4d2-b4a2a7d422f5.png">